setup.sh

```diff
@@ -167,6 +167,7 @@ then
         cat temp
         cat temp >> $log
         echo "Kindly kill above running instance(s) else change port number in config file, then continue to run this script." | tee -a $log
+        echo "By Using: kill `sudo lsof -t -i:#PORT_NUM`"
```

Comment:
Kindly merge this statement to the earlier message like,

kill above running instance by issuing : "kill `sudo lsof -t -i:#PORT_NUM`"
else
change port number in config file, then continue with this script